Hestia/app/src/androidTest/java/com/rugged/application/hestia/ActivatorAndActivatorStateTest.java

```diff
@@ -1,97 +0,0 @@
-package com.rugged.application.hestia;
-
-import android.support.test.runner.AndroidJUnit4;
-
-import org.junit.Before;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-
-import hestia.backend.Activator;
-import hestia.backend.ActivatorState;
-
-import static org.junit.Assert.*;
-
-@RunWith(AndroidJUnit4.class)
-public class ActivatorAndActivatorStateTest {
```

Comment:
Last sprint we worked hard on removing all coupling between the front end and the back end of the client side code, this was needed to easily integrate the error message that will be coming in the next iteration. 

To do this a lot of the code base was changed. Old test were no longer viable for this new structure, so we deleted them. We didn't have enough time to write new tests for the different structure. 

As you saw we had already some testing methods prepared to implement. Hopefully this will bring back the code coverage to at least the 50% mark.